zk/syncer/l1_syncer.go

```diff
@@ -302,7 +302,7 @@ func (s *L1Syncer) getLatestL1Block() (uint64, error) {
 }
 
 func (s *L1Syncer) queryBlocks() error {
-	startBlock := s.lastCheckedL1Block.Load()
+	startBlock := s.lastCheckedL1Block.Load() + 1
 
 	log.Debug("GetHighestSequence", "startBlock", s.lastCheckedL1Block.Load())
```

Comment:
Let's also make this use start block and not the atomic

Reply:
Done